bin/start-docker.sh

```diff
@@ -26,8 +26,6 @@ eval "$(docker-machine env dinghy)"
 BIN_DIR=$(pushd `dirname "$0"` 2>&1 > /dev/null && pwd && popd  2>&1 > /dev/null)
 BASE_DIR=$(pushd "${BIN_DIR}/../" 2>&1 > /dev/null && pwd && popd  2>&1 > /dev/null)
 
-${BIN_DIR}/copy-protobuf.sh
```

Comment:
git rm these files? I don't think these are needed anymore

Reply:
ah yeah i think I removed them in the other draft PR I have open, but since they are no longer referenced here I should remove them. thanks for pointing this out!